app.js

```diff
@@ -204,9 +203,6 @@ if (useDocumentation || onlyDocumentation === 'true') {
     extended: true,
   }));
```

Comment:
Delete this whole section?

Reply:
Let's do it in a follow-up PR, removing the password stuff